tr/cs-221-states-models.md

```diff
@@ -0,0 +1,980 @@
+
+&#10230; [İki oyunculu sıfır toplamlı oyun ― Her durumun tamamen gözlendiği ve oyuncuların sırayla oynadığı bir oyundur. Aşağıdaki gibi tanımlanır:, bir başlangıç durumu sstart, s durumunda gerçekleştirilebilecek olası eylemler Actions(s), s durumunda a eylemi gerçekleştirildiğindeki ardıllar Succ(s,a), bir bitiş durumuna ulaşılıp ulaşılmadığı IsEnd(s), s bitiş durumunda etmenin elde ettiği fayda Utility(s), s durumunu kontrol eden oyuncu Player(s)]
+
+<br>
+
+
+**101. Remark: we will assume that the utility of the agent has the opposite sign of the one of the opponent.**
+
+&#10230; Not: Oyuncu faydasının işaretinin, rakibinin faydasının tersi olacağını varsayacağız.
+
+<br>
+
+
+**102. [Types of policies ― There are two types of policies:, Deterministic policies, noted πp(s), which are actions that player p takes in state s., Stochastic policies, noted πp(s,a)∈[0,1], which are probabilities that player p takes action a in state s.]**
+
+&#10230; [Politika türleri ― İki tane politika türü vardır:, Belirlenimci politikalar, πp(s) olarak gösterilir, p oyuncusunun s durumunda gerçekleştirdiği eylemler., Olasılıksal politikalar, πp(s,a)∈[0,1] olarak gösterilir, p oyuncusunun s durumunda a eylemini gerçekleştirme olasılıkları.]
+
+<br>
+
+
+**103. Expectimax ― For a given state s, the expectimax value Vexptmax(s) is the maximum expected utility of any agent policy when playing with respect to a fixed and known opponent policy πopp. It is computed as follows:**
+
+&#10230; En yüksek beklenen değer(Expectimax) ― Belirli bir s durumu için, en yüksek beklenen değer, Vexptmax(s), sabit ve bilinen bir rakip politikasına,πopp, göre oynarken, bir oyuncu politikasının en yüksek beklenen faydasıdır. En yüksek beklenen değer(Expectimax) aşağıdaki gibi hesaplanmaktadır:
+
+<br>
+
+
+**104. Remark: expectimax is the analog of value iteration for MDPs.**
+
+&#10230; Not: En yüksek beklenen değer(Expectimax), MDP'ler için değer yinelemenin analog halidir.
+
+<br>
+
+
+**105. Minimax ― The goal of minimax policies is to find an optimal policy against an adversary by assuming the worst case, i.e. that the opponent is doing everything to minimize the agent's utility. It is done as follows:**
+
+&#10230; En küçük-en büyük (minimax) ― En küçük-enbüyük (minimax) politikaların amacı en kötü durumu kabul ederek, diğer bir deyişle, rakip, oyuncunun faydasını en aza indirmek için her şeyi yaparken, rakibe karşı en iyi politikayı bulmaktır. En küçük-en büyük(minimax) aşağıdaki şekilde yapılır:
+
+<br>
+
+
+**106. Remark: we can extract πmax and πmin from the minimax value Vminimax.**
+
+&#10230; Not: πmax ve πmin değerleri, en küçük-en büyükten,Vminimax, elde edilebilir.
```

Comment:
Not: πmax ve πmin değerleri, en küçük-en büyük olan Vminimax'dan elde edilebilir.